app/util/sanitize.js

```diff
@@ -0,0 +1,12 @@
+const regHeaders = (data) => {
+    var reg = new RegExp('([a-zA-Z|-|,|s]*)');
+    return reg.exec(data);
+};
+
+const regOrigin = (data) => {
```

Comment:
regHeaders --> validatereOrigin

Reply:
good unit test coverage needed, ideally with some real values acceptable and imaginative bad values

Reply:
Updated